evil-winrm.rb

```diff
@@ -916,8 +916,8 @@ def set_cache(n_path, paths)
 
     def normalize_path(str)
         p_str = str || ""
```

Comment:
If this first line is only meant to handle `nil` inputs, we could instead unconditionally coerce to strings:
```ruby
def normalize_path(string)
  Regexp.escape(string.to_s.gsub('\\', '/'))
end
```
This'll work because `nil.to_s` => `""`.

Reply:
Updated in 9fc1f2f.